srcpkgs/kodi-addon-pvr-zattoo/template

```diff
@@ -14,5 +14,9 @@ maintainer="teldra <teldra@rotce.de>"
 license="GPL-2.0-only"
 homepage="https://github.com/rbuehlma/pvr.zattoo"
 distfiles="https://github.com/rbuehlma/pvr.zattoo/archive/${version}-${_kodi_release}.tar.gz"
-checksum=19de7bc58bcf37bbcff5ad2cb2f095cb15a4217f9abec5c9ff30d0489dab89d2
+checksum=9c3dc2ed9f8c39bcc4f5798439b85f2bdae1a427ffeb1b9d18aa5e2caca29f4e
 nocross="depends on kodi-platform"
+
+do_check() {
+	:
+}
```

Comment:
Same as the other, use `make_check` and explain why you disabled.

Reply:
Done